scripts/yul_coverage.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+FAILED=()
+SUCCESS=()
+test_file()
+{
+  local SOL_FILE
+  local OUTPUT
+
+  SOL_FILE=${1}
+  OUTPUT=$(solc --ir "${SOL_FILE}" 2>&1 || true)
```

Comment:
This will use system-wide `solc` (unless you have `build/solc` in your `PATH` for some reason). Looking at other scripts I see that they tend use paths relative to `ROOT_DIR`. Not perfect since this assumes you have your `build/` directory inside the source tree (for example I do not) but still better in my opinion than silently using the wrong `solc` executable.

Reply:
yes, thats true, its more consistent to use `solc` from `build/`.

Reply:
`solc` can now be defined by setting the environment variable `SOLC`, it will default to `solc`